x-pack/plugins/observability/public/pages/slos/components/slo_list_item.tsx

```diff
@@ -172,97 +119,13 @@ export function SloListItem({
-            isOpen={isActionsPopoverOpen}
-          >
-            <EuiContextMenuPanel
-              size="m"
-              items={[
-                <EuiContextMenuItem
-                  key="view"
-                  icon="inspect"
-                  onClick={handleViewDetails}
-                  data-test-subj="sloActionsView"
-                >
-                  {i18n.translate('xpack.observability.slo.item.actions.details', {
-                    defaultMessage: 'Details',
-                  })}
-                </EuiContextMenuItem>,
-                <EuiContextMenuItem
-                  key="edit"
-                  icon="pencil"
-                  disabled={!hasWriteCapabilities}
-                  onClick={handleEdit}
-                  data-test-subj="sloActionsEdit"
-                >
-                  {i18n.translate('xpack.observability.slo.item.actions.edit', {
-                    defaultMessage: 'Edit',
-                  })}
-                </EuiContextMenuItem>,
-                <EuiContextMenuItem
-                  key="createRule"
-                  icon="bell"
-                  disabled={!hasWriteCapabilities}
-                  onClick={handleCreateRule}
-                  data-test-subj="sloActionsCreateRule"
-                >
-                  {i18n.translate('xpack.observability.slo.item.actions.createRule', {
-                    defaultMessage: 'Create new alert rule',
-                  })}
-                </EuiContextMenuItem>,
-                <EuiContextMenuItem
-                  key="manageRules"
-                  icon="gear"
-                  disabled={!hasWriteCapabilities}
-                  onClick={handleNavigateToRules}
-                  data-test-subj="sloActionsManageRules"
-                >
-                  {i18n.translate('xpack.observability.slo.item.actions.manageRules', {
-                    defaultMessage: 'Manage rules',
-                  })}
-                </EuiContextMenuItem>,
-                <EuiContextMenuItem
-                  key="clone"
-                  disabled={!hasWriteCapabilities}
-                  icon="copy"
-                  onClick={handleClone}
-                  data-test-subj="sloActionsClone"
-                >
-                  {i18n.translate('xpack.observability.slo.item.actions.clone', {
-                    defaultMessage: 'Clone',
-                  })}
-                </EuiContextMenuItem>,
-                <EuiContextMenuItem
-                  key="delete"
-                  icon="trash"
-                  disabled={!hasWriteCapabilities}
-                  onClick={handleDelete}
-                  data-test-subj="sloActionsDelete"
-                >
-                  {i18n.translate('xpack.observability.slo.item.actions.delete', {
-                    defaultMessage: 'Delete',
-                  })}
-                </EuiContextMenuItem>,
-              ]}
-            />
-          </EuiPopover>
+          <SLOItemActions
```

Comment:
I would rename to `SloItemActions` to stick to the existing naming convention of using `Slo` instead of `SLO` for the react components.